### ISSUE TITLE
`cd bento; python setup.py install --user` gives an error

### ISSUE THREAD

**Issue by cdeil** [cournape/Bento#139]
It seems the `--user` option to install Bento is broken.

http://cournape.github.io/Bento/html/install.html#setuptools-based-installation-deprecated

```
$ python setup.py install --user
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: option --user not recognized
```

**Comment by cdeil** [comment]
Please also document how to install Bento in the site-packages that `--user` refers to with setuptools, e.g. here:
http://cournape.github.io/Bento/html/install.html#bento-based-installer

**Comment by rgommers** [comment]
Maybe setup.py is not supported at all anymore? http://cournape.github.io/Bento/html/install.html says it's deprecated.

**Comment by cdeil** [comment]
I don't want to install Python packages into the system `site-packages`, only in my home directory.
And the recommended `./bentomaker install` doesn't accept a `--user` option and it's not clear to me from the docs if there is an equivalent option.

If `python setup.py install` is deprecated it should still work, but it doesn't.

It's pretty sad that a Python package that's supposed to help with installation of other Python packages can't make it's own installation simple.

If this project is completely unmaintained at the moment, maybe best to say so and not point people to  it from the `numpy`, `scipy` and `scikit-image` build instructions?

**Comment by rgommers** [comment]
Agreed that that should be possible. @cournape how do you want to handle this? I assume `python setup.py install --user` should be made to work (again), so `pip` can be used?

**Comment by rgommers** [comment]
Can be made to work at the moment with `python configure --sitedir=path/to/local/sitedir` I believe. But `pip` doesn't know about that.

**Comment by cdeil** [comment]
I saw that Bento has a [bentomaker configure --prefix](http://cournape.github.io/Bento/html/tutorial.html#bentomaker) option.

But that's much less convenient than a `--user` option ... make me manually figure out where the `site-packages` in my home folder is first ... why?

**Comment by rgommers** [comment]
Agreed, a lot less convenient. This is similar to how you build Python, not a Python package. Need @cournape to explain this one.
